examples/tv-app/linux/include/application-basic/ApplicationBasicManager.h

```diff
@@ -41,10 +41,8 @@ class ApplicationBasicManager : public ApplicationBasicDelegate
         CopyString(mVendorName, sizeof(mVendorName), szVendorName);
         mVendorId = vendorId;
         CopyString(mApplicationVersion, sizeof(mApplicationVersion), szApplicationVersion);
-        mProductId                          = productId;
-        static const uint16_t kTestVendorId = 456; // CI test cases require this vendor id
```

Comment:
this change is only here so that I can force a failure in CI in order to track down the test code which is hardcoding this value. I will ultimately remove this change.